numpy/random/tests/test_random.py

```diff
@@ -352,40 +356,41 @@ def test_bytes(self):
         np.testing.assert_equal(actual, desired)
 
     def test_shuffle(self):
```

Comment:
The description is not the issue with debugging, it's the context when you enable ``--pdb-failure`` or ``--ipdb-failure`` - this is what many of us use for debugging failed tests.